airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py

```diff
@@ -70,6 +70,7 @@
     VendorSalesReports,
     VendorTrafficReport,
     XmlAllOrdersDataByOrderDataGeneral,
+    ListTransactions
```

Comment:
Does it make sense to name this stream `FinanceTransactions` instead of `ListTransactions`? Feels like `ListTransactions` is a little ambiguous. Thoughts?

Reply:
@zerobearing2  I think I could change the stream name to match the Finance streams. 

Do you mind providing me with amazon seller account credentials temporarily so I can generate the schema directly from airbyte UI ? The current json response schema (manually done) is the source of the error. That would really help. I pointed this out on slack.

Reply:
Do you mind temporarily providing me with credentials so I can generate the exact schema I wish to get ? Its quite large and complex to do manually

Reply:
@zerobearing2

Reply:
@Harmaton I can't provide any live credentials, sorry. I was just researching how to provide sandbox access. I'm unframiliar with seller centeral sandbox, so not sure exactly how to grant access yet. However, it seems the sandbox just responds with a "mocked" response. The finance mocked response comes from github here: https://github.com/amzn/selling-partner-api-models/blob/main/models/finances-api-model/finances_2024-06-19.json

Is that something you could use? I will continue looking at how to provide temporary sandbox credentials and reply when I have something to share.